scraper/src/config/config_loader.py

```diff
@@ -18,10 +17,15 @@
 from .browser_handler import BrowserHandler
 
 try:
+    from urllib.parse import urlparse, unquote_plus
+except ImportError:
     from urlparse import urlparse
     from urllib import unquote_plus
+
+try:
+    from collections.abc import OrderedDict  # Python 3
 except ImportError:
-    from urllib.parse import urlparse, unquote_plus
+    from collections import OrderedDict  # Python 2.7
 
 
 class ConfigLoader(object):
```

Comment:
```suggestion
class ConfigLoader:
```

Reply:
Will require another PR (should impact a lot of other part of the codebase). Thanks for sharing the good practice.